tests/common/snappi/snappi_fixtures.py

```diff
@@ -517,3 +521,361 @@ def cvg_api(snappi_api_serv_ip,
     yield api
     if getattr(api, 'assistant', None) is not None:
         api.assistant.Session.remove()
+
+
+def snappi_dut_base_config(duthost_list,
+                           tgen_ports,
```

Comment:
Why we have both tgen_ports and snappi_ports here?

Reply:
tgen_ports argument has been removed